dist/index.js

```diff
@@ -417,7 +417,8 @@ function run() {
         try {
             const envPath = core.getInput('path');
             const envExamplePath = core.getInput('example-path');
-            const variables = parseEnv_1.default(envPath, envExamplePath);
+            const allowEmpty = core.getInput('allow-empty-values');
```

Comment:
What do you think about casting it to a boolean right here and throwing a meaningful error if that fails. This way we get proper error handling and can use it as a boolean for the rest of the script.